client/rest/build.gradle

```diff
@@ -51,9 +52,9 @@ dependencies {
   api "commons-codec:commons-codec:${versions.commonscodec}"
   api "commons-logging:commons-logging:${versions.commonslogging}"
   api "org.slf4j:slf4j-api:${versions.slf4j}"
-  runtimeOnly "org.bouncycastle:bc-fips:${versions.bouncycastle_jce}"
-  runtimeOnly "org.bouncycastle:bctls-fips:${versions.bouncycastle_tls}"
-  runtimeOnly "org.bouncycastle:bcutil-fips:${versions.bouncycastle_util}"
+  fipsOnly "org.bouncycastle:bc-fips:${versions.bouncycastle_jce}"
```

Comment:
@beanuwave I am a bit confused by the scope of the latest changes: we used to have compile / runtime / api / implementation / testImplementation scopes but for FIPS it is now reduced to fipsOnly / testfipsOnly (==testImplementation which is fine) only. Could you please clarify the intent here? Thank you.